### PR TITLE
added Canvas and .Net-core dependency

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by pescobar** [comment]
travis complains that the download url line is too long but how can I workaround that?

https://github.com/easybuilders/easybuild-easyconfigs/pull/7716/files#diff-36d17a226d1183e0c1a494a17e73a220R18

File: easybuild/easyconfigs/n/Net-core/Net-core-2.1.8.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+version = '2.1.8'
+
+homepage = 'https://dotnet.microsoft.com/'
+description = """.NET Core is a free and open-source managed computer software framework for the Windows,
+ Linux, and macOS operating systems .NET Core fully supports C# and F# and partially supports Visual Basic"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = [
+    'https://download.visualstudio.microsoft.com/download/pr/927cbf41-157f-488b-ac7d-32034defebc3/41b011418f795d5a76bf930ad57a055b/'
```
Comment (akesandgren) [inline easybuild/easyconfigs/n/Net-core/Net-core-2.1.8.eb]: You could put that url into a variable, which you create in two steps, or do
'https://download.visualstudio.microsoft.com/download/pr/' +
'927cbf41-157f-488b-ac7d-32034defebc3/41b011418f795d5a76bf930ad57a055b/'
which should? be ok according to pep8

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2006.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0928e769cae8920e61fb5121a4caad73 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2406.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/08a0ac909317c37534f1cbdcaa578ebe for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**